DependencyInjection/EzSystemsTweetFieldTypeExtension.php

```diff
@@ -18,8 +18,16 @@ class EzSystemsTweetFieldTypeExtension extends Extension implements PrependExten
 {
     public function prepend(ContainerBuilder $container)
     {
-        $config = Yaml::parse(file_get_contents(__DIR__ . '/../Resources/config/ez_field_templates.yml'));
-        $container->prependExtensionConfig('ezpublish', $config);
+        $configDirectoryPath = __DIR__ . '/../Resources/config';
+        $configFiles = [
+            'ez_platformui' => $configDirectoryPath . '/yui.yml',
+            'ezpublish' => $configDirectoryPath . '/ez_field_templates.yml'
+        ];
+
+        foreach ($configFiles as $extensionName => $configFilePath) {
```

Comment:
IMHO the look doesn't make this more readable.

Maybe something like:
```php
$this->prependYamlConfigFile($container, 'ez_platformui', $configDirectoryPath . '/yui.yml');
$this->prependYamlConfigFile($container, 'ezpublish', $configDirectoryPath . '/ez_field_templates.yml');
```

with
```php
private function prependYamlConfigFile(ContainerBuilder $container, $configFilePath)
{
    $config = Yaml::parse(file_get_contents($configFilePath));
    $container->prependExtensionConfig($extensionName, $config);
}
```